torch_xla/distributed/cluster.py

```diff
@@ -382,6 +383,39 @@ def add_service_worker(tpu_name):
 
     return workers
 
+  def get_tpuvm_client_workers(self):
+    workers = []
+
+    def add_tpuvm_client_worker(tpu_name):
```

Comment:
Could we just make this method take in a list of `service_workers` and basically just translate  those into `client_workers`? I don't want to block PR much longer so I can approve if needed.

Reply:
I merge it with the `get_service_workers`, it might be a bit cleaner that way